src/style.css

```
@@ -0,0 +1,137 @@
+html {
+  margin: 0;
+  width: 0;
+}
+
+.card {
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-picture {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 36px;
+}
+
+.card-product__description {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px;
+  color: #060b35;
+}
+
+.card-product__code {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  line-height: 14px;
+  margin-left: 16px;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.card-rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 14px;
+  margin-left: 16px;
+  margin-bottom: 24px;
+}
+
+p {
+  margin: 0;
+}
+
+.card-rate__stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card-rate__stars > :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars-4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card-rate__feedback {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  margin-right: 16px;
+  color: #060b35;
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card-price__number {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 18px;
+  margin-right: 16px;
+  color: #060b35;
+}
+
+.card-buy {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  margin: 16px;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  color: #fff;
+  border-radius: 5px;
+}
```

better put to goup modifiers and split by empty line
```suggestion
.card-buy {
  display: flex;
  justify-content: center;
  align-items: center;
  
  margin: 16px;
  width: 166px;
  height: 40px;
  
  font-family: Roboto, sans-serif;
  font-style: normal;
  font-weight: bold;
  font-size: 14px;
  line-height: 16px;
  color: #fff;
  text-transform: uppercase;
  text-align: center;
  
  background: #00acdc;
  border-radius: 5px;
}
```
see [recommendations](http://codeguide.academy/html-css.html#css-order)